README.md

```diff
@@ -84,6 +84,18 @@ int main()
 }
 ```
 
+Compatibility Guidelines
+========================
+
+The users if cppzmq are expected follow the guidelines below for breakage-free behaviour when upgrading cppzmq to newer versions (non-exhaustive list):
+
+* Do not depend on any macros defined in cppzmq unless explicitly exported.
+
+The following macros are exported: `CPPZMQ_VERSION`, `CPPZMQ_VERSION_MAJOR`, `CPPZMQ_VERSION_MINOR`, `CPPZMQ_VERSION_PATCH`. The following macros may be defined to opt in/out of behaviour:
```

Comment:
These are two distinct aspects, and should be separated better. One is lead to think that the second sentence is related to exported/public macros as well, but on the contrary, it is related to macros that are imported/consumed. This should be made clearer.